src/sentry/tagstore/snuba/backend.py

```diff
@@ -689,6 +689,11 @@ def get_group_ids_for_search_filter(
         # instead.
         raise NotImplementedError
 
+    def delete_tag_key(self, project_id, key):
+        from sentry import eventstream
+        eventstream_state = eventstream.start_delete_tag(project_id, key)
+        eventstream.end_delete_tag(eventstream_state)
```

Comment:
We use `start_` and `end_` to:

1. Allow us to do Sentry-side logic in-between the two in the future, as other messages do.
2. Maintain a relatively unified API with the other messages, at very low cost.

Reply:
Ah right, duh.